pkg/config/setup/config.go

```diff
@@ -1236,6 +1236,7 @@ func autoconfig(config pkgconfigmodel.Setup) {
 	config.BindEnvAndSetDefault("container_exclude_logs", []string{})
 	config.BindEnvAndSetDefault("container_exclude_stopped_age", DefaultAuditorTTL-1) // in hours
 	config.BindEnvAndSetDefault("ad_config_poll_interval", int64(10))                 // in seconds
+	config.BindEnv("ad_allowed_env_vars")
```

Comment:
Please add a default to any new settings. a `nil` slices would to the trick. Types are required is we want to have a schema at some point.